Makefile

```diff
@@ -166,7 +166,7 @@ build-local-image-%:
 		.
 
 # Build all images locally
-build-local-images: $(addprefix build-image-,$(IMAGE_NAMES))
+build-local-images: $(addprefix build-local-image-,$(IMAGE_NAMES))
```

Comment:
I'm guessing you didn't intend to commit this?

Reply:
I couldn't get a local build to work without this, thought it was necessary, unless I've got to also specify some other bits and bobs.